src/app/components/cards/PostNotificationsList.jsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import TimeAgoWrapper from 'app/components/elements/TimeAgoWrapper';
+import LoadingIndicator from 'app/components/elements/LoadingIndicator';
+import { actions as fetchDataActions } from 'app/redux/FetchDataSaga';
+import Callout from 'app/components/elements/Callout';
+
+class PostNotificationsList extends React.Component {
```

Comment:
@gl2748 can we re-use the existing NotificationsList, or at least the bulk of its rendering logic? It still needs quite a bit of polishing and ideally we don't have to modify two copies.

Reply:
@roadscape I've now done this.